lib/matplotlib/backends/backend_agg.py

```diff
@@ -594,6 +594,29 @@ def print_tif(self, filename_or_obj, *, pil_kwargs=None):
 
     print_tiff = print_tif
 
+    @_check_savefig_extra_args
+    def print_webp(self, filename_or_obj, *, pil_kwargs=None):
```

Comment:
Are _all_ these pillow save formats the same?  If so, I wonder if we just want a generic `print_pillow(format=pillow_format)` rather than adding these one-by-one?

Reply:
As far as I can tell, it is the same for all the fully supported formats- BMP, DDS, DIB, EPS, GIF, ICNS, ICO, IM, JPEG, JPEG 2000, MSP, PCX, PNG, PPM, SGI, SPIDER, TGA, TIFF, WebP, XBM. 
I think adding a generic `print_pillow` does make a lot of sense especially if we want to support all of these formats. The tests however will have to be created for each format, as they all have their specific options that need to be tested.

Reply:
Thats the approach I would take, but maybe its more complex than that...

Reply:
Maybe we should leave it for a different PR then?